<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Technofit</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_device_btn">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(28, 113, 216);
	border: 1px solid rgb(255, 255, 255);
	border-radius: 10px;
	color:rgb(255, 255, 255);
}
QPushButton:hover
{
	background-color: rgb(26, 95, 180);
	border: 1px solid rgb(255, 255, 255);
	border-radius: 10px;
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Add new device</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1031</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="add_layout"/>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>31</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="pulse_pic">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);
border:none;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="O2_pic">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);
border:none;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="temprature_pic">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);
border:none;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ECG_pic">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);
border:none;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
